package echo;

option cc_generic_services = true;

message EchoRequest
{
	required string message = 1;
};

message EchoResponse
{
	required string response = 1;
};

service EchoService
{
	rpc Echo(EchoRequest) returns (EchoResponse);
};
